website/src/content/docs/docs/migrate-swagger/faq/breakingchange.md

```diff
@@ -2,16 +2,48 @@
 title: Resolving Swagger Breaking Change Violations
 ---
 
-The Swagger Converter will not be able to accurately represent every part of every API in TypeSpec. This document
-outlines some common changes you might need to make to a converted TypeSpec to make it conform to your existing service API and  
-pass check-in validations.
+The Swagger Converter cannot perfectly represent every aspect of every API in TypeSpec. This document outlines common changes you may need to make to a converted TypeSpec to ensure compatibility with your existing service API and to pass check-in validations.
 
-## Migrate ARM Specs
+## Migrating ARM Specifications
 
-### Changing the Names of Request Payload Parameters
+### Customizing Route for Action (POST) Operations
 
-For operations with non-empty request bodies (PUT, POST, PATCH), the TypeSpec operation templates provide a default name for the
-request parameter corresponding to the request payload. You can use augment decorators to make changes to this parameter, and other parts of the operation signature.
+By default, the last segment of a POST action operation route is the operation name. For example:
+
+```tsp
+move is ArmResourceActionSync<Employee, MoveRequest, MoveResponse>;
+```
+
+This produces the route:  
+`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContosoProviderHub/employees/{employeeName}/move`
+
+To customize the action segment, use the `@action` decorator:
+
+```tsp
+@action("customizedAction")
+move is ArmResourceActionSync<Employee, MoveRequest, MoveResponse>;
+```
+
+This produces the route:  
+`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContosoProviderHub/employees/{employeeName}/customizedAction`
+
+By default, the `@action` decorator converts the segment to camel case. If you want to preserve the exact casing, use `@Rest.Private.actionSegment`:
```

Comment:
We should not include this advice.  ARM APIs should be case insensitive.  Service teams should use the camelCase version and ensure it is supported by their service.

Reply:
+1 with Mark